bower.json

```diff
@@ -14,7 +14,7 @@
     "tests"
   ],
   "dependencies": {
-    "o-typography": "^5.7.5",
+    "o-typography": "#v5.8.0",
```

Comment:
```suggestion
    "o-typography": "^5.8.0",
```

Reply:
This is temporary :) Waiting for v5.8.0 to be released

https://github.com/Financial-Times/o-layout/pull/51#issue-243768945
> Still to come: Update o-typography dependency, update MIGRATION guide, refactor some more.